algorithms/src/sorting/Kokkos_SortPublicAPI.hpp

```diff
@@ -264,7 +117,7 @@ void sort(ViewType view, size_t const begin, size_t const end) {
 
   Kokkos::fence("Kokkos::sort: before");
 
-  if (view.extent(0) == 0) {
+  if (view.extent(0) <= 0) {
```

Comment:
```suggestion
  if (view.extent(0) <= 1) {
```

Reply:
fixed all of them, thank you - that was a bad slip !